### DISCUSSION TITLE
Integration official rpc with flume

### DISCUSSION THREAD

**Discussion by hackfisher** [ringdao-community-guild/discussion#107] [Integration]
Easier and first choice might be self host, we may need @fewensa or someone from community to find the setup-with-loadbalancer docs
https://blog.openrelay.xyz/flume/

Ref:
https://github.com/openrelayxyz/cardinal-flume
https://blog.rivet.cloud/polygon/flumelight/

**Comment by nblogist** [comment]
@fewensa lmk if you need help in anything!

**Comment by hackfisher** [comment]
@fewensa says it is not open? He prefers hypersync now.
